national_rail/load.py

```diff
@@ -0,0 +1,113 @@
+""" Loads incident data into the RDS database. """
+from os import environ
+from dotenv import load_dotenv
+
+from psycopg2 import connect
+from psycopg2.extensions import connection, cursor
+from psycopg2.extras import RealDictCursor
+
+
+def get_connection() -> connection:
+    """ Retrieves connection and returns it. """
+    load_dotenv()
+    return connect(
+        user=environ['DB_USERNAME'],
+        password=environ['DB_PASSWORD'],
+        host=environ['DB_IP'],
+        port=environ['DB_PORT'],
+        dbname=environ['DB_NAME']
+    )
+
+
+def get_cursor(conn: connection) -> cursor:
+    """ Retrieves cursor and returns it. """
+    return conn.cursor(cursor_factory=RealDictCursor)
+
+
+def upload_incident(conn: connection, incident: dict) -> int:
+    """ Takes an incident and uploads to the database, returning the incident id. """
+
+    query = """
+        INSERT INTO incident (incident_number, creation_time, incident_start, incident_end, is_planned, incident_summary,
+            incident_description, incident_uri, affected_routes)
+            VALUES (%s, %s, %s, %s, %s, %s, %s, %s, %s) 
+        RETURNING incident_id;
+    """
+
+    cur = get_cursor(conn)
+    cur.execute(query, (
+        incident["incident_number"],
+        incident["creation_time"],
+        incident["start_time"],
+        incident["end_time"],
+        incident["is_planned"],
+        incident["summary"],
+        incident["description"],
+        incident["uri"],
+        incident["routes_affected"],
+
```

Comment:
It might be best to put this in the try except block in the scenario that this execute statement doesn't work.

Reply:
good idea